src/appleseed.cli/CMakeLists.txt

```diff
@@ -59,11 +59,11 @@ add_executable (appleseed.cli
     ${appleseed.cli_sources}
 )
 
-if (CMAKE_SYSTEM_NAME STREQUAL "Linux")
+if (UNIX AND NOT WIN32 AND NOT APPLE)
```

Comment:
Actually it looks like Cygwin [does support](https://cygwin.com/ml/crossgcc/2009-02/msg00019.html) `$ORIGIN` in rpath, so the check can be spelled as `UNIX AND NOT APPLE`.

I'll try to cook up a patch for top level to get `APPLESEED_USE_RPATH` support.